k8s/aat/software-engineering/draft-store/draft-store-service.yaml

```
@@ -12,9 +12,11 @@ spec:
   chart:
     repository: https://hmcts.azurecr.io/helm/v1/repo/
     name: draft-store-service
-    version: 0.0.2
+    version: 0.0.3
   values:
     java:
+      replicas: '2'
```

these aren't environment variables so they should be their actual values
```suggestion
      replicas: 2
```